fs/dafilesrv/dafilesrv.cpp

```diff
@@ -788,12 +788,16 @@ int main(int argc,char **argv)
             return ret;
 #endif
     }
+#ifndef _CONTAINERIZED
     {
         Owned<IComponentLogFileCreator> lf = createComponentLogFileCreator(logDir.str(), "DAFILESRV");
         lf->setCreateAliasFile(false);
         lf->setMaxDetail(TopDetail);
         lf->beginLogging();
     }
+#else
+        setupContainerizedLogMsgHandlerFromConfig();
```

Comment:
formatting: indentation

Reply:
Similar to comment in sasha - dafilesrv does not use the new config files yet (doesn't call loadConfiguration), so if this were called it would throw an exception in queryComponentConfig().
But dafilesrv isn't running in any of our containers yet, so it's fine.
When/if it is, we will need to ensure it uses loads a config.